src/Stash.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018, 2019 Transposit Corporation. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Transposit } from ".";
+import { KeyValuePair, MutableKeyValueStore } from "./KeyValueStore";
+
+export class Stash implements MutableKeyValueStore<any> {
```

Comment:
I don't dislike the idea of having separate namespaces for functionality. I do want to understand what it'll look like when we release 2.0.0.

Right now, I know we have:
- operation.{run}()
- stash.{list, get, put, delete}()

What else will we have?

Reply:
env.{get, getBuiltin}()
user_settings.{get, put, remove}()
Possibly listKeys on those too? Otherwise I'd remove listKeys from the KeyValueStore interface.

Reply:
Unless there is a really good reason to deviate, I think we should try to be consistent with the graal bindings in execution. Concretely this means we shouldn't have stash.delete() - this is done with stash.put(<key>, null). Likewise, there's no listKeys for env and user_settings.

Reply:
note also that getBuiltin may not be relevant in the context of the SDK, at least right now. Today, it returns an object with two fields:
- appUrl: the hosted app url, which I assume folks hosting their front end on netlify aren't using?
- requestId: only makes sense inside an execution request